docs/architecture-overview.md

```diff
@@ -51,7 +51,7 @@ Licensing: [Apache License 2.0](https://github.com/JodaOrg/joda-time/blob/master
 
 
 ### Room
-Room is a persistence library that provides an abstraction layer over SQLite. In contrary against the EncryptedSharedPreferences Room is used for storing more complex data.
+Room is a persistence library that provides an abstraction layer over SQLite. In contrary against the SharedPreferences Room is used for storing more complex data.
 
```

Comment:
Awkward English sentence :)  Maybe we could correct that as well
As opposed to SharedPreferences, Room..... or
Contrary to SharedPreferences, Room...

Reply:
Thanks, I felt weird about it too :)